.github/workflows/behat-tests.yml

```diff
@@ -0,0 +1,207 @@
+      COMPOSER_NO_AUDIT: 1
+      COMPOSER_PROCESS_TIMEOUT: 0
+      WP_VERSION: 6.8.3
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: Setup PHP ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: mysqli, intl, mbstring
+          tools: composer:v2
+          coverage: none
+
+      - name: Install MySQL client
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y mysql-client
+
+      - name: Compute DB name per PHP
+        run: |
+          PV="${{ matrix.php }}"
+          SAFE="${PV/./}"
+          echo "DB_NAME=wp_test_${SAFE}" >> "$GITHUB_ENV"
+
+      - name: Create database
+        run: |
+          echo "Creating database $DB_NAME"
+          mysql -h 127.0.0.1 -uroot -proot -e "DROP DATABASE IF EXISTS \`$DB_NAME\`; CREATE DATABASE \`$DB_NAME\` CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;"
+
+      - name: Prepare WP Core + Tests (all logic lives in the script)
+        env:
+          DB_NAME: ${{ env.DB_NAME }}
+        run: bash bin/phpunit-bootstrap.sh
+
+      - name: Run PHPUnit
+        env:
+          DB_NAME: ${{ env.DB_NAME }}
+          XDEBUG_MODE: off
+        run: |
+          if [ -x vendor/bin/phpunit ]; then
+            vendor/bin/phpunit
+          else
+            PHPV=$(php -r 'echo PHP_MAJOR_VERSION.".".PHP_MINOR_VERSION;')
+            if [ "$PHPV" = "8.4" ]; then
+              curl -fsSL https://phar.phpunit.de/phpunit-10.phar -o phpunit.phar
+            else
+              curl -fsSL https://phar.phpunit.de/phpunit-9.phar -o phpunit.phar
+            fi
+            php ./phpunit.phar
+          fi
+
+  # DO NOT TOUCH — finalised & well-tested
+  behat:
+    name: Behat (SSP ${{ matrix.ssp }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ssp: ["1.18.0", "2.0.0", "2.4.0"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+          submodules: recursive
+          lfs: false
+
+      - name: Setup PHP 8.2
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "8.2"
+          tools: composer:v2
+          coverage: none
+
+      - name: Install Terminus
+        uses: pantheon-systems/terminus-github-actions@v1
+        with:
+          pantheon-machine-token: ${{ secrets.TERMINUS_TOKEN }}
+
+      - name: SSH agent (Pantheon)
+        uses: webfactory/ssh-agent@v0.9.0
+        with:
+          ssh-private-key: ${{ secrets.SITE_OWNER_SSH_PRIVATE_KEY }}
+          log-public-key: true
+
+      - name: Configure SSH (no strict host key checking)
+        run: |
+          mkdir -p ~/.ssh
+          printf "Host *\n  StrictHostKeyChecking no\n  UserKnownHostsFile=/dev/null\n" > ~/.ssh/config
+
+      # As in CircleCI: hard-set site id
```

Comment:
Remove comment referring to Circle. Unnecessary since, after this, we won't be using Cirlce, so this reference will become dated (and therefore confusing) in the future.

```suggestion
```